rep-0147.rst

```diff
@@ -312,22 +313,22 @@ A standard way to communicate goals will also be important following the mavlink
   uint8 FRAME_GLOBAL_TERRAIN_ALT = 11
 
   uint16 type_mask
-  uint16 IGNORE_LATITUDE = 1	# Position ignore flags
-  uint16 IGNORE_LONGITUDE = 2
-  uint16 IGNORE_ALTITUDE = 4
-  uint16 IGNORE_VX = 8	# Velocity vector ignore flags
-  uint16 IGNORE_VY = 16
-  uint16 IGNORE_VZ = 32
-  uint16 IGNORE_AFX = 64	# Acceleration/Force vector ignore flags
-  uint16 IGNORE_AFY = 128
-  uint16 IGNORE_AFZ = 256
-  uint16 FORCE = 512	# Force in af vector flag
-  uint16 IGNORE_YAW = 1024
-  uint16 IGNORE_YAW_RATE = 2048
+  uint16 TYPE_MASK_IGNORE_LATITUDE = 1	# Position ignore flags
+  uint16 TYPE_MASK_IGNORE_LONGITUDE = 2
+  uint16 TYPE_MASK_IGNORE_ALTITUDE = 4
+  uint16 TYPE_MASK_IGNORE_VX = 8	# Velocity vector ignore flags
+  uint16 TYPE_MASK_IGNORE_VY = 16
+  uint16 TYPE_MASK_IGNORE_VZ = 32
+  uint16 TYPE_MASK_IGNORE_AFX = 64	# Acceleration/Force vector ignore flags
+  uint16 TYPE_MASK_IGNORE_AFY = 128
+  uint16 TYPE_MASK_IGNORE_AFZ = 256
+  uint16 TYPE_MASK_FORCE = 512	# Force in af vector flag
+  uint16 TYPE_MASK_IGNORE_YAW = 1024
+  uint16 TYPE_MASK_IGNORE_YAW_RATE = 2048
```

Comment:
As far as I can tell, these don't currently have that prefix: https://github.com/mavlink/mavros/blob/2e20c1cbe62fe2bdec51ae6f63bdd740dee39cb7/mavros_msgs/msg/GlobalPositionTarget.msg .  So please put these back to what they were.

Reply:
They do not, but I was trying to follow the commonly adopted convention that the prefix matches these "enum" values. Do you think  it is a requirement to exactly follow the Mavlink originations of all the fields? ABI will already be broken because the namespace changes.

Reply:
Ok! I'll chat with the macros maintainers and see if they are interested.

Reply:
Note - this is actually a bitmask, and the bitmask fields in the mavlink protocol include the word typemask. 
https://mavlink.io/en/messages/common.html#POSITION_TARGET_TYPEMASK

Reply:
I removed the TYPE_MASK prefix change from this PR, and will come back to it when ready.